src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/SpecifyStringComparison.cs

```diff
@@ -208,5 +213,24 @@ private static ParameterInfo GetParameterInfo(INamedTypeSymbol type, bool isArra
         {
             return ParameterInfo.GetParameterInfo(type, isArray, arrayRank, isParams);
         }
+
+        private static IEnumerable<IMethodSymbol> GetAccessibleMethodsWithSameNameAsTargetMethod(
+            IInvocationOperation invocationExpression,
+            IMethodSymbol targetMethod,
+            SemanticModel? semanticModel)
```

Comment:
`Operation.SemanticModel` is guaranteed to be non-null for analyzer callbacks. You can remove the null check for semanticModel.
```suggestion
            SemanticModel semanticModel)
```